### DISCUSSION TITLE
疫情衝擊下，香港基層家庭及兒童面臨心理健康挑戰

### DISCUSSION THREAD

**Discussion by Guagua1234** [1101-GS4719-C/Python-Community-GS4719-C#1139] [General]
https://www.thenewslens.com/article/177877
我覺得新冠疫情爆發後，關於兒童心理健康的關注越來越多。特別是在香港這樣高壓力的生活環境下，兒童心理健康也受到了越來越多的關注。我發現香港"救助兒童會"發布的研究報告，世界衛生組織估計有近40%的香港中小學生可能有困難集中精神、焦躁和失眠等問題。我覺得特別是在新冠疫情爆發時,學童在家上網課，失去平常在學校習慣的人際互動，心理的調適需要更多時間，也會以電玩或沉迷網絡、電視等方式轉移壓力，家長對此或不了解，導致越來越多的矛盾。兒童乃至家長的精神健康都有了問題。

**Comment by 110302550** [comment]
在疫情的衝擊底下，人們的生活都變了樣，最難受的還是這些小孩，他們的童年被困在家哩，人與人的溝通也變得很少，周遭環境的氣氛也變得非常緊張，希望疫情趕快過去，恢復原本的生活
